### DISCUSSION TITLE
Not able to view Job's Logs in Github Actions

### DISCUSSION THREAD

**Discussion by prakash-cts** [community/community#102737] [Actions]
### Select Topic Area

Bug

### Body

Hi All,

I am getting this error while trying to view logs in Github Action jobs. Please help in resolving this.

<img width="486" alt="Screenshot 2024-01-30 at 1 00 21 PM" src="https://github.com/community/community/assets/69976672/6d21c986-d564-4f53-80ec-26b4e9496975">

**Comment by mickeygousset** [comment]
Have you tried re-running the workflow with debugging turned on?

**Reply by prakash-cts** [reply]
I am able to view the logs when debugging is turned on. But why this error message though. I was able to view those logs earlier and the error is inconsistent. Seems if I use a external shell script in one of the jobs, I get this error. Will update if I am able to find the cause of the issue.

**Reply by mickeygousset** [reply]
Is your workflow in a public repo where we can see it? or can you post it here?
